components/integration/production/base/manager_resources_patch.yaml

```diff
@@ -11,10 +11,10 @@ spec:
         resources:
           limits:
             cpu: 600m
-            memory: 1500Mi
+            memory: 2048Mi
```

Comment:
+1, but how did you decide on these values?

Reply:
It's in line with what the other services have set (e.g. see #4380), and we've been seeing memory spikes of up to 1.7 G when starting or resyncing the service (which is the most memory demanding part of the lifecycle).

Reply:
thank you!